src/js/helpers/searchIndexBuilder.js

```diff
@@ -0,0 +1,73 @@
+const searchIndexBuilder = function(pages) {
+
+  const searchIndex = []
+
+  pages.forEach( page =>{
+
+    if (page.node.janisUrls.length > 0) {
+
+      const janisPages = [
+        "locationpage",
+        "eventpage",
+        "departmentpage",
+        "topiccollectionpage"
+      ]
+
+      const janisBasePagesWithTopics = [
+        "servicepage",
+        "guidepage",
+        "informationpage",
+        "officialdocumentpage",
+        "formcontainer"
+      ]
+
+      // Checking if the page is one of our Janis Pages without topics.
+      // - If so, Add it to our search index.
+      janisPages.forEach( pageType => {
+        if (page.node[pageType]) {
+          searchIndex.push({
+            title: page.node[pageType].title,
+            janisUrls: page.node.janisUrls,
+            pageType: pageType,
+            summery: page.node[pageType].mission ?? page.node[pageType].shortDescription ?? ""
```

Comment:
I'm guessing this is supposed to be summary

Reply:
its also summery on the joplin side